<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1531</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QFrame" name="frame_graphicsView_integration1DView"/>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_plotType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Summed Single Pt. Counts (horizontal)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Summed Single Pt. Counts (vertical)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2-theta FWHM Model</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_plot">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Scan">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scan number (single Pt.) to plot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_rewindPlot">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>|&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_forwardPlot">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;&gt;|</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_exportToMovie">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Export To Movie</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>(Gaussian) Integration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ROI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_integratePeaks">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step 4: Estimate/calculate peak intensities with Gaussian model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Integrate Peaks</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_exportIntensityToFile">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Export To File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_roiList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>160</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sum Single Pt. Counts     </string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_retrieveFWHM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>160</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step 3: Use the loaded 2theta-sigma curve to calculate FWHM of the peaks in the table by interpolation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Calculate FWHM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_load2thetaSigmaFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>160</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step 2: Load a (at least) two column file for 2theta - (Gaussian) sigma curve,&lt;/p&gt;&lt;p&gt;which will be used to calculate/estimate the peak's FWHM for single-measurement peaks&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Load FWHM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pushButton_exportIntensityToTable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Export To Table</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_integrteDetectorCounts">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>160</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step 1: Integrate detector counts by given ROI for all the (selected) scans in the table above&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Sum Counts</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_integrateDirection">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Counts sum/integration direction.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Vertical: detector counts are summed in the vertical direction (along Y-axis).  Output will be x vs counts.&lt;/p&gt;&lt;p&gt;Horizontal: detector counts are summed in the horizontal direction (along X-axis).  Output will be y vs counts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>Horizontal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vertical</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_fitPeaks">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This check box is used with two push buttons&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;1. &amp;quot;Sum Counts&amp;quot;: when checked,  peaks will be fitted with Gaussian&lt;/p&gt;&lt;p&gt;2. &amp;quot;Plot&amp;quot; (to the right side): when checked, calculated data (from peak fitting) will be plot agains the original data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Fit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_tableView_summary"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1531</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Table View</string>
    </property>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDe_select_All"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Gaussian_Sigma_File"/>
    <addaction name="actionLoad_Peak_Info_File"/>
   </widget>
   <widget class="QMenu" name="menuView_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh_ROI_List"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCalculate_FWHM"/>
    <addaction name="actionDefine_2theta_FWHM_Function"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView_2"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDe_select_All">
   <property name="text">
    <string>De-select All</string>
   </property>
  </action>
  <action name="actionLoad_Gaussian_Sigma_File">
   <property name="text">
    <string>Load Gaussian-Sigma File</string>
   </property>
  </action>
  <action name="actionLoad_Peak_Info_File">
   <property name="text">
    <string>Load Peak Info File</string>
   </property>
  </action>
  <action name="actionRefresh_ROI_List">
   <property name="text">
    <string>Refresh ROI List</string>
   </property>
  </action>
  <action name="actionCalculate_FWHM">
   <property name="text">
    <string>Calculate FWHM</string>
   </property>
  </action>
  <action name="actionDefine_2theta_FWHM_Function">
   <property name="text">
    <string>Define $\2theta$-FWHM Function</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SinglePtIntegrationTable</class>
   <extends>QTableView</extends>
   <header>HFIR_4Circle_Reduction/hfctables.h</header>
  </customwidget>
  <customwidget>
   <class>SinglePtIntegrationView</class>
   <extends>QGraphicsView</extends>
   <header>HFIR_4Circle_Reduction/integratedpeakview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
